test/mocks/Erc4626.sol

```diff
@@ -22,6 +25,26 @@ contract Erc4626 {
   /// @dev the last amount withdraw was called with, reciever -> WithdrawArgs
   mapping (address => WithdrawArgs) public withdrawCalled;
 
+  function asset() public view returns (address) {
+    return assetReturn;
+  }
+
+  function assetReturns(address a) public {
+    assetReturn = a;
+  }
+
+  // NOTE using the solc feature to squelch the compiler nag about unused vars by not having a var name
+  function convertToAssets(uint256) public view returns (uint256) {
+    return convertToAssetsReturn;
+  }
+
+  // NOTE we refrain from a convertToAssetsCalled here as it's a hard coded 1e26
+  // also we would lose our ability to have it interfaced as `view`
+
+  function convertToAssetsReturns(uint256 a) public {
+    convertToAssetsReturn = a;
+  }
```

Comment:
Please add natspec docs for these methods.

Reply:
this is a mock, but sure.